xbmc/interfaces/builtins/SkinBuiltins.cpp

```diff
@@ -100,6 +102,47 @@ static int SetAddon(const std::vector<std::string>& params)
   return 0;
 }
 
+/*! \brief Select and set a skin bool setting.
+ *  \param params The parameters.
+ *  \details params[0] = Names of skin settings.
+ */
+static int SelectBool(const std::vector<std::string>& params)
+{
+  std::vector<std::pair<std::string, std::string>> settings;
+
+  CGUIDialogSelect* pDlgSelect = (CGUIDialogSelect*)g_windowManager.GetWindow(WINDOW_DIALOG_SELECT);
+  pDlgSelect->Reset();
+  pDlgSelect->SetHeading(CVariant{g_localizeStrings.Get(424)});
+
+  for (unsigned int i = 0 ; i < params.size() ; i++)
```

Comment:
> as this is more a coding style issue, not programmatically wrong

I know, but why not encourage using the c++11 coding style. Its only a small change...